contrib/RL_out_sharp.lua

```diff
@@ -0,0 +1,208 @@
+--[[
+
+  DESCRIPTION
+    RL_out_sharp.lua - Richardson-Lucy output sharpening using GMic
+
+    This script provides a new target storage "RL output sharpen". 
+    Images exported will be sharpened using GMic (RL deblur algorithm)
+    
+  USAGE
+    * require this script from main lua file
+    * in lua preferences, select the GMic cli executable
```

Comment:
After you select the executable, you have to restart darktable so that the darktablerc file, with the new preference gets read, otherwise the preference isn't available.

My preference is to do a df.check_if_bin_exists() then add the df.executable_path_widget() if it doesn't so the user can add the executable and then proceed to use the script without having to restart darktable.  If the executable does exist, then don't show the widget.

Reply:
yes, I considered that option.
Personally, I prefer not to clutter the GUI with a widget that you use only once and forget
Moreover, df.check_if_bin_exists() adds a separator and a label that takes some space in the panel.
Maybe I can say to restart DT in the instructions, is that OK ?

Reply:
You would use the widget the first time and then it wouldn't appear because the value is already set.  

But, adding restart to the instructions is fine too.